lib/common/config.go

```diff
@@ -17,8 +17,9 @@ type Config struct {
 	BlockTime         time.Duration
 	BlockTimeDelta    time.Duration
 
-	TxsLimit int
-	OpsLimit int
+	TxsLimit    int
+	OpsLimit    int
+	TxPoolLimit int
```

Comment:
In `run.go`, `txPoolLimit` is `unit64`. How about set `TxPoolLimit` to be same, `uint64` instead of `int`?

Reply:
I want to use `int`,  because of `pool.Len() int`. And lrucache use  the size  as `int`. https://godoc.org/github.com/hashicorp/golang-lru#New